src/js/angular/core/interceptors/authentication.interceptor.js

```diff
@@ -8,6 +8,14 @@ angular.module('graphdb.framework.core.interceptors.authentication', [
             return {
                 'request': function(config) {
                     const headers = config.headers || {};
+
+                    // When using OpenID, during authentication process, the additional headers modification must be skipped
```

Comment:
Do repository headers have to be skipped?

Reply:
Yes, those requests are to the Auth service and not GDB, so the repo information is not needed.